components/cam/src/control/scamMod.F90

```diff
@@ -581,7 +581,7 @@ subroutine setiopupdate
           write(iulog,*) 'nstep = ',get_nstep()
           write(iulog,*) 'ncdate (E3SM date) =',ncdate,' ncsec=',ncsec
           write(iulog,*) 'next_date (IOP file date) =',next_date_print,' &
-	                  next_sec=',next_sec_print
+	                  &next_sec=',next_sec_print
```

Comment:
Just curious, does the following work (entire string in the next line)?
```
write(iulog,*) 'next_date (IOP file date) =',next_date_print,&
	                  'next_sec=',next_sec_print
```

Reply:
Yes, that also works.

Reply:
This looks cleaner.

Reply:
OK, I will update it with Jayesh's suggestion.